server/__init__.py

```diff
@@ -76,11 +76,10 @@ def create_app():
     app.register_blueprint(ranking.bp)
     app.register_blueprint(redirects.bp)
     app.register_blueprint(tools.bp)
-    from routes.api import (browser as browser_api, chart, choropleth, place as
+    from routes.api import (browser as browser_api, choropleth, place as
```

Comment:
shall we rename all the "choropleth" as "map"?

Reply:
I can, but would I want to rename choropleth stuff in the place page to map as well? Or we could keep this api as choropleth and the name "map" would just be specific to the map explorer tool which plots a choropleth map

Reply:
+1 to keeping map as just the product name

Reply:
Change all the place page would be too many.. Yea, non-place change sounds good. This is not a must have, can do after all the feature is done.